daemon/create_unix.go

```diff
@@ -42,7 +43,15 @@ func createContainerPlatformSpecificSettings(container *Container, config *runco
 			return fmt.Errorf("cannot mount volume over existing file, file exists %s", path)
 		}
 
-		v, err := createVolume(name, config.VolumeDriver)
+		// if a bind is not specified then default to the local volume driver
+		var volumeDriver string
```

Comment:
you can set `volumeDriver := volume.DefaultDriverName` by default and avoid the `else`.